### ISSUE TITLE
`rtt_resend_factor`

### ISSUE THREAD

**Issue by simbleau** [cBournhonesque/lightyear#386]
https://github.com/cBournhonesque/lightyear/blob/303a2f00201a0e55bdd84fb940e955ee47387187/lightyear/src/channel/builder.rs#L199

Docs say this is a Duration. Is this wrong? It's an f32.

**Comment by Nul-led** [comment]
Yea docs are wrong, its a factor, not the actual duration

**Comment by cBournhonesque** [comment]
Yep, it's a multipler of the current RTT estimate, the docs need to be updated
